spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -405,15 +422,14 @@ definitions:
               Phase biases corrections for a
               satellite being tracked.
 
-
  - MSG_SSR_STEC_CORRECTION:
     id: 0x05EB
     short_desc: Slant Total Electron Content
     desc: |
       The STEC per space vehicle, given as polynomial approximation for
-      a given grid.  This should be combined with SSR-GriddedCorrection
+      a given grid area.  This should be combined with MSG_SSR_GRIDDED_CORRECTION
```

Comment:
not sure why we need "area".  We have a grid that has been defined.  Of course a grid covers an area; I feel like this is tautological and thus more confusing.

Reply:
sure i can drop that